product_docs/docs/epas/14/deployment_options/epas_inst_linux/install_details/rpm_packages.mdx

```diff
@@ -896,404 +163,42 @@ apt list edb*
-    systemctl daemon-reload
-    ```
-
--   Now, the `/opt/edb` location has been labeled correctly with the context, use the following command to start the service:
-
-    ```text
-    systemctl start edb-as-14
-    ```
-
-## Starting multiple postmasters with different clusters
-
-You can configure EDB Postgres Advanced Server to use multiple postmasters, each with its own database cluster. The steps required are version specific to the Linux host.
-
-**On RHEL/CentOS 7.x or RHEL/Rocky Linux/AlmaLinux 8.x**
-
-The `edb-as14-server-core` RPM for version 7.x | 8.x contains a unit file that starts the EDB Postgres Advanced Server instance. The file allows you to start multiple services, with unique `data` directories and that monitor different ports. You must have `root` access to invoke or modify the script.
-
-The example that follows creates an EDB Postgres Advanced Server installation with two instances; the secondary instance is named `secondary`:
-
--   Make a copy of the default file with the new name. As noted at the top of the file, all modifications must reside under `/etc`. You must pick a name that is not already used in `/etc/systemd/system`.
-
-    ```text
-    cp /usr/lib/systemd/system/edb-as-14.service /etc/systemd/system/secondary-edb-as-14.service
-    ```
-
--   Edit the file, changing `PGDATA` to point to the new `data` directory that you will create the cluster against.
-
--   Create the target `PGDATA` with user `enterprisedb`.
-
--   Run `initdb`, specifying the setup script:
-
-    ```text
-    /usr/edb/as14/bin/edb-as-14-setup initdb secondary-edb-as-14
-    ```
-
--   Edit the `postgresql.conf` file for the new instance, specifying the port, the IP address, TCP/IP settings, etc.
-
--   Make sure that new cluster runs after a reboot:
-
-    ```text
-    systemctl enable secondary-edb-as-14
-    ```
-
--   Start the second cluster with the following command:
-
-    ```text
-    systemctl start secondary-edb-as-14
-    ```
-
-## Creating an EDB Postgres Advanced Server repository on an isolated network
-
-You can create a local repository to act as a host for the EDB Postgres Advanced Server RPM packages if the server on which you wish to install EDB Postgres Advanced Server (or supporting components) cannot directly access the EDB repository. This is a high-level listing of the steps requires; modify the process for your individual network.
-
-To create and use a local repository, you must:
-
--   Use `yum` or `dnf` to install the `epel-release`, `yum-utils`, and `createrepo` packages.
-
-    On RHEL or CentOS 7.x:
-
-    ```bash
-    yum install epel-release
-    yum install yum-utils
-    yum install createrepo
-    ```
-
-    On RHEL or Rocky Linux or AlmaLinux 8.x:
-
-    ```bash
-    dnf install epel-release
-    dnf install yum-utils
-    dnf install createrepo
-    ```
-
--   Create a directory in which to store the repository:
-
-    ```text
-    mkdir /srv/repos
-    ```
+### Updating an RPM installation
```

Comment:
This section shouldn't be here. It should be in the Upgrading section. Need to compare with the existing topic that is already there.

Reply:
Agreed, but we just decided to shuffle the content of installation topics hence I didn't remove it from here. We can pull it into upgrading the topic now or later when we do a full restructure.